pkg/component/observability/plutono/plutono.go

```diff
@@ -231,7 +231,7 @@ func (p *plutono) computeResourcesData(ctx context.Context) (*corev1.ConfigMap,
 	}
 	dataSourceConfigMap := &corev1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "plutono-datasources",
+			Name:      "plutono-datasources" + dataSourcesKeySuffix,
```

Comment:
Is it possible to cover this with unit test to make sure we won't regress it in the future?